### PR TITLE
Fixes #1097 - Validate title attribute to prevent display Tooltips with empty messages

### LINKED ISSUE #1097: Tooltip is displayed with empty content
I think the tooltip has never been displayed with empty content

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

**Comment by diegonvs** [comment]
@carloslancha @matuzalemsteles, makes sense?

File: packages/clay-tooltip/demos/index.html [hunk 1]
```diff
@@ -20,6 +20,10 @@
 		<h1>Tooltip</h1>
 	</div>
 
+	<button style="margin:50px;position:absolute;top:500px;left:50px;" title="">
+		<h1>Should not render ClayTooltip!</h1>
```
Comment (matuzalemsteles) [inline packages/clay-tooltip/demos/index.html]: hey @diegonvs you can remove the `h1` tag. Just not to get that big. 🙂

Comment (diegonvs) [reply packages/clay-tooltip/demos/index.html]: I completely forgot this, sorry 🙏

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
@diegonvs you need to run `npm run format`, travis is failing ;)

**Comment by carloslancha** [comment]
Resent https://github.com/liferay/clay/pull/1695

**Pull request closed without merging**